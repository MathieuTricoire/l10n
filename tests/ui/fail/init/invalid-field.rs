l10n::init!({
    format: || {}
});

fn main() {}
