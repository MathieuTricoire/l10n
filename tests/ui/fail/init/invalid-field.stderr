error: invalid field (expected: "formatter", "transform", "use_isolating" or "functions")
 --> tests/ui/fail/init/invalid-field.rs:2:5
  |
2 |     format: || {}
  |     ^^^^^^
